browser/ui/brave_pages.cc

```diff
@@ -24,8 +24,8 @@ void ShowBraveRewards(Browser* browser) {
   ShowSingletonTabOverwritingNTP(browser, GURL(kRewardsPageURL));
 }
 
-void ShowBraveAdblock(Browser* browser) {
-  ShowSingletonTabOverwritingNTP(browser, GURL(kBraveUIAdblockURL));
+void ShowBraveAdblock(Profile* profile) {
```

Comment:
@bridiver all these functions use `Browser*` to display the requested page in the exact browser window, NOT in the last active window. Passing `Profile*` here will switch those function to look for the last active browser window and load the requested page there instead of the explicit browser window the caller asks for.